lib/suspenders/app_builder.rb

```diff
@@ -237,6 +237,15 @@ def configure_action_mailer
       action_mailer_host "production", %{ENV.fetch("HOST")}
     end
 
+    def configure_active_job
+      config = <<-RUBY
+
+    config.active_job.queue_adapter = :delayed_job
+      RUBY
+
+      inject_into_class 'config/application.rb', 'Application', config
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.